cmd/blockchaincmd/create.go

```diff
@@ -193,6 +195,11 @@ func createBlockchainConfig(cmd *cobra.Command, args []string) error {
 		return errors.New("flags --evm,--custom are mutually exclusive")
 	}
 
+	if nonSOV {
+		if createFlags.proofOfAuthority || createFlags.proofOfStake || createFlags.poaValidatorManagerOwner != "" {
```

Comment:
do we have some kind of any func over a list of boolean? we should have. probably in the flag package,